cmake/projects/re2/hunter.cmake

```diff
@@ -152,6 +152,50 @@ hunter_add_version(
     e88e72adc5dd18d00922a9c7ddf6e056e29a3d06
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    re2
+    VERSION
+    2022.06.01
+    URL
+    https://github.com/google/re2/archive/2022-06-01.tar.gz
+    SHA1
+    547c9a5bbb4133bed1aae47037bba1bdade6ec9d
+)
+
+hunter_add_version(
+    PACKAGE_NAME
+    re2
+    VERSION
+    2022.12.01
+    URL
+    https://github.com/google/re2/archive/2022-12-01.tar.gz
+    SHA1
+    8146fb81e2b8988a455f2f7291c7a8a4001e55a6
+)
+
+hunter_add_version(
+    PACKAGE_NAME
+    re2
+    VERSION
+    2023.02.01
```

Comment:
non-blocking: do we need all versions available? Has this a great benefit I'm missing?

Reply:
@hjmallon any thoughts about this wquestion? (or this PR in general as you seem to use this package)

Reply:
I dont have a need for all the versions, only the most recent I guess. I can see that it is useful to have the most recent version without abseil though.

Reply:
@NeroBurner I included the interim versions because it was easy to do so. I tested all versions locally.

Reply:
then thanks for the effort, merging